### DISCUSSION TITLE
Draw pictures

### DISCUSSION THREAD

**Discussion by jgfioruci** [russhughes/gc9a01_mpy#31] [Q&A]
I'm having troubles on converting an image to a bitmap and show it on the display, I'd like to know how to do it properly.

**Comment by russhughes** [comment]
Take a look at this example: https://russhughes.github.io/gc9a01_mpy/examples/bluemarble.html. If your image is a jpg you can do something like this: https://russhughes.github.io/gc9a01_mpy/examples/jpg_test.html
